dockerfiles/che/Dockerfile

```diff
@@ -9,14 +9,11 @@
 # Variables in `COPY --from=` is not supported see https://github.com/moby/moby/issues/34482
 # this is workaround to handle that.
 ARG CHE_DASHBOARD_ORGANIZATION=quay.io/eclipse
-ARG CHE_DASHBOARD_NEXT_ORGANIZATION=quay.io/eclipse
-ARG CHE_DASHBOARD_VERSION=next
-ARG CHE_DASHBOARD_NEXT_VERSION=next-react
+ARG CHE_DASHBOARD_VERSION=next-react
 ARG CHE_WORKSPACE_LOADER_ORGANIZATION=quay.io/eclipse
 ARG CHE_WORKSPACE_LOADER_VERSION=next
```

Comment:
We don't need a workspace loader anymore. Could you remove it?

Reply:
Good point! I will remove it and test

Reply:
I've taken a look and it leads to removing a few maven modules, java classes, adapting release scripts and I would like to move it out of the current PR scope. So, I've created a dedicated issue for that https://github.com/eclipse/che/issues/18785

Reply:
> then why is the CHE_DASHBOARD_VERSION=next-react instead of =next ?

it's not. Probably you just see the outdated diff displayed here. Let me know if you see the same on https://github.com/eclipse/che/pull/18549/files